openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/ChangeLogs.tsx

```diff
@@ -25,10 +25,15 @@ const ChangeLogs = ({ data }: Props) => {
 
   return (
     <div>
-      {logKeys.map((log, index) => (
-        <div className="mb-4" key={index}>
+      {logKeys.map((log) => (
+        <div className="mb-4" key={log}>
```

Comment:
Don't we have any unique identifier here?

Reply:
That log is a unique key, so that will work.

Reply:
Hy we can't have longer text as the key
Will log always be of few words?

Reply:
Yes, the log will be of a few words only.